scss/bitstyles.scss

```diff
@@ -54,8 +54,8 @@
 // Organisms ///////////////////////////////////
 
 @import 'bitstyles/organisms/modal/';
-@import 'bitstyles/organisms/expander';
 @import 'bitstyles/organisms/navbar/';
+@import 'bitstyles/organisms/table/';
```

Comment:
Why is this still called "organisms" while the stories where changed to "ui"?

Reply:
There’s a plot between the “UI” documentation and the components beneath — the UI-level docs are mostly examples of pre-built components that often compose a bunch of atoms, organisms, and utilities. They don’t necessarily nor often correspond to a separate organism.

Tables and navbars need their own CSS to work (and some utilities etc), so they exist as organisms in the CSS, and like all the CSS classes they’re documented in storybook, in a manner which shows the classes and child classes, but not necessarily how best to use them. The documentation on tables at the UI level shows how to use the tables in a roughly real-world instance, but doesn’t show you examples of each of the child elements, classes etc.

That’s because each of those layers is aimed at a different person and/or use case. Does that make sense, any downsides or problems that you see? I’m very sure that I’ll need to do a cleanup round to be sure that structure is implemented well across all examples & classes, as my ideas on this have developed quite a lot as I’ve been working on the changes

Reply:
Makes sense, thanks for explaining 🙇